package.json

```diff
@@ -14,7 +14,7 @@
     "test:cypress": "start-server-and-test dev 3000 cy:run",
     "danger": "danger",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook -c .storybook -o ../project-idea-reviewer-nextjs-storybook/docs"
```

Comment:
Not a huge fan of touching directories outside of the project root... Can't we just add the storybook build output to the `.gitignore` if we don't want it to be added to the repo?

Reply:
I understand your reticence.  However, I'm going to make an executive decision here: while this is not ideal, it has the advantage that (a) it's scalable to the entire class (b)  we already have the repos set up to handle it (because of the tooling in the ucsb-cs-github-linker).    So while your objection is noted, this ship has already sailed. 

It isn't just a matter of not committing it to the repo.  It's a matter of getting it published on GitHub pages.   While that could be managed with  branches, our students are not yet solid enough in their GitHub knowledge to make that workable.